app/views/finders/show.html.erb

```diff
@@ -55,7 +55,7 @@
         >
         <div id="js-search-results-info" data-module="remove-filter" class="result-info <%= "govuk-!-margin-bottom-0" if result_set_presenter.has_sort_options %>">
           <div class="govuk-grid-row">
-            <div class="govuk-grid-column-one-third">
+            <div class="govuk-grid-column-one-half">
```

Comment:
This seems unrelated to the work described - nothing wrong with it, but could you include it in the PR description if it's intentional to include here.